modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/GridCacheMultithreadedFailoverAbstractTest.java

```diff
@@ -69,6 +70,7 @@
 /**
  * Base test for all multithreaded cache scenarios w/ and w/o failover.
  */
+@Ignore("FAILED! Abstract")
```

Comment:
Does it have any running subclasses? If so, it does not need to be included in suites itself. If it doesn't, let's make it not abstrace?

Reply:
@alamar,
As @timoninmaxim mentioned in the email such classes have no inheritors.
I've rechecked for this class and may confirm this is true.

Reply:
Let's un-abstract it if it does not have inheritors.

Reply:
Un-abstarct it, add jira link.